backend/app/api/routes/login_modern.py

```
@@ -0,0 +1,333 @@
+            "is_setup_complete": getattr(user, 'is_setup_complete', True)
+        }
+
+        new_access_token = ModernSecurityManager.create_access_token(
+            subject=user.id,
+            additional_claims=additional_claims
+        )
+
+        # 缓存新token
+        try:
+            from datetime import datetime, timedelta
+            expires_at = datetime.now(timezone.utc) + timedelta(minutes=15)
```

Ruff (0.12.2)

Undefined name `timezone`

(F821)